task09-reactjs/src/components/AllCards.js

```diff
@@ -0,0 +1,77 @@
+import classes from './AllCards.module.css'
+import CardList from './cards/CardList'
+import chuckLogo from './images/logo/chuck-norris.png'
+import powerLogo from './images/logo/power-rangers.png'
+import kameLogo from './images/logo/kame.png'
+import capsuleLogo from './images/logo/capsule-corp.jpg'
+import raptorLogo from './images/logo/raptor-squad.jpg'
+import donutLogo from './images/logo/donuts-security.png'
+import shieldLogo from './images/logo/Shield-logo.png'
+import certified from './images/certified.svg'
+const DATA = [

```

Comment:
Move the data to another file to keep the component clean

Reply:
Done